### PR TITLE
Find your machine's IP address.

### PR DESCRIPTION
**Pull request by rishibaghel25**
It would help new Linux users to find your machine's IP address

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 1]
```diff
@@ -57,6 +57,19 @@ A SODA release conists of various projects which have their own release cycles a
 
 Set the environment variable `HOST_IP` by using the steps below. 
 
+
+To set the HOST_IP environment variable, you first need to find your machine's IP address.
+Use the following command to display network interface information:
+
+```bash
+ifconfig
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:65]: ifconfig is not available on Ubuntu 18.04 and above, by default. Alternatively `ip -4 addr` will give the available network connections.

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 2]
```diff
@@ -57,6 +57,19 @@ A SODA release conists of various projects which have their own release cycles a
...
+```
+Look for the IP address associated with the network interface you are using (e.g. eth0 or wlan0) and find the inet address,
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:67]: When using VirtualBox this is different. usually enps0**

**Review by anvithks — CHANGES_REQUESTED** [review]
I am not sure whether we need to include such detailed steps to find out the IP address. Just a suggestion on how it can be done (which command) should be enough.
@skdwriting @joseph-v Thoughts?

**Comment by rishibaghel25** [comment]
Many people who are new to Linux may not be familiar with basic commands like cd,ls,nano,kill..... As a student who has used Linux before, I have some idea of linux . However, there are many students who are unfamiliar with Linux commands. Therefore, I suggest that providing detailed steps would be helpful for beginners. Even I'm also getting many errors during installation 😅

**Pull request closed without merging**